beets/ui/commands.py

```diff
@@ -1017,6 +1018,12 @@ def import_func(lib, opts, args):
     metavar='ID',
     help=u'restrict matching to a specific metadata backend ID'
 )
+import_cmd.parser.add_option(
+    u'--set-field', dest='set_fields', action='callback',
```

Comment:
This is a little bit of bikeshedding, but what do you think about the slightly simpler option name `--set`? In beets, the only thing you can really "set" is fields, so having "field" in the name feels a teensy bit redundant.